templates/core/features/index.html

```diff
@@ -340,31 +319,87 @@ <h3>Snapcraft</h3>
   </div>
 </section>
 
-<section class="p-strip--light">
-  <div class="row u-equal-height">
-    <div class="col-5 u-hide--medium u-hide--small u-align--center">
-          {{ image (
-            url="https://assets.ubuntu.com/v1/c4b290c8-Contact+us.svg",
-            alt="",
-            width="281",
-            height="200",
-            hi_def=True,
-            loading="lazy"
-            ) | safe
-          }}
+<section class="p-section">
+  <div class="row--50-50">
+    <hr class="p-rule" />
+    <div class="col">
+      <h2>Tamper-resistant and hardened</h2>
     </div>
-    <div class="col-7">
-      <h2>Secure your devices</h2>
-      <p>Get in touch with a Ubuntu security expert to discuss the advanced security requirements of your application.</p>
-      <p>
-        <a href="/core/contact-us?product=core-features" class="p-button--positive js-invoke-modal">Get in touch</a>
-      </p>
+    <div class="col">
+      <div class="p-section--shallow">
+        <p class="p-heading--5">Every aspect of the system is checked and verified</p>
+        <p>You need to know your software is pristine; not just for installation, but for the lifetime of the device. Immutable packages and persistent digital signatures mean Ubuntu Core can verify any software component at any time, to guard against corruption and attack.</p> 
+        <p>Comply with cybersecurity requirements and build a reliable business.</p>
+      </div>
+      <hr />
+      <a href="/engage/ubuntu-core-security-audit">Read an independent security audit of Ubuntu Core&nbsp;&rsaquo;</a>
     </div>
   </div>
 </section>
 
-  <!-- Set default Marketo information for contact form below-->
-  <div class="u-hide" id="contact-form-container" data-form-location="/shared/forms/interactive/internet-of-things" data-form-id="1266" data-lp-id="2166" data-return-url="https://ubuntu.com/core/thank-you" data-lp-url="https://pages.ubuntu.com/things-contact-us.html">
+<section class="p-section">
+  <div class="row--50-50">
+    <hr class="p-rule" />
+    <div class="col">
+      <h2>Learn more about Ubuntu Core</h2>
+    </div>
+    <div class="col">
+      <ul class="p-list--divided">
+        <li class="p-list__item u-no-padding--top u-no-padding--bottom">
+          <div class="row--50-50">
+            <div class="col">
+              <p class="p-heading--5">Docs</p>
```

Comment:
```suggestion
              <h3 class="p-heading--5">Docs</h3>
```
I believe these should be h3s